js/tests/loadtest.ts

```diff
@@ -44,173 +46,136 @@ let TRANSFERS_TOTAL = 100;
         syncWallets.push(syncWallet);
 
         // Create wallets
-        var i = 1;
-        while (i < CLIENTS_TOTAL) {
-            let ew = ethers.Wallet.createRandom().connect(ethersProvider);
-            let sw = await Wallet.fromEthWallet(
+        for (let i = 1; i < CLIENTS_TOTAL; i++) {
+            const ew = ethers.Wallet.createRandom().connect(ethersProvider);
+            const sw = await Wallet.fromEthSigner(
                 ew,
                 syncProvider,
-                ethProxy
+                ethProxy,
             );
             ethWallets.push(ew);
             syncWallets.push(sw);
-
-            i++;
         }
 
         // Deposits
-        i = 0;
-        while  (i < CLIENTS_TOTAL) {
-            await depositEther(ethWallets[0], syncWallets[i], DEPOSIT_AMOUNT);
-            i++;
-        }
+        await deposit(ethWallets[0], syncWallets[0], ["ETH", ERC20_TOKEN], initDepositAmount);
+        await transfer(syncWallets[0], syncWallets, ["ETH", ERC20_TOKEN], depositAmount);
 
         // Transfers
-        i = 0;
-        while(i < TRANSFERS_TOTAL) {
-            let number1 = randomClientNumber();
-            var number2 = randomClientNumber();
-            while (number2 == number1) {
-                number2 = randomClientNumber();
-            }
-            let client1 = syncWallets[number1];
-            let client2 = syncWallets[number2];
+        let promises = [];
+
+        for (const wallet of syncWallets) {
+            promises.push(transfer(wallet, syncWallets, ["ETH", ERC20_TOKEN], depositAmount.div(TRANSFER_DIVISOR)));
+        }
+        let results = await Promise.all(promises.map(reflect));
 
-            await transferEther(client1, client2, TRANSFER_AMOUNT);
-            i++;
+        let failedPromises = results.filter((p) => p.status === "rejected");
+        for (const promise of failedPromises) {
+            console.log(`Failed transfer: ${promise.reason}`);
         }
 
         // Withdraws
-        i = 0;
-        while  (i < CLIENTS_TOTAL) {
-            await withdrawEther(ethWallets[i], syncWallets[i]);
-            i++;
+        promises = [];
+        results = [];
+        failedPromises = [];
+
+        for (const wallet of syncWallets) {
+            const i = syncWallets.indexOf(wallet);
+            promises.push(withdraw(wallet, ethWallets[i], ["ETH", ERC20_TOKEN]));
+        }
+        await Promise.all(promises.map(reflect));
```

Comment:
`Promise.all` will reject if one of them will reject -- it is not acceptable. Use something like `Promise.allSettled()`